navigation/navigation-common/src/commonMain/kotlin/androidx/navigation/NoOpNavigator.kt

```diff
@@ -20,7 +20,7 @@ import androidx.core.bundle.Bundle
 
 /** A [Navigator] that only supports creating destinations. */
 @RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
-public expect class NoOpNavigator : Navigator<NavDestination> {
+public expect class NoOpNavigator() : Navigator<NavDestination> {
```

Comment:
`actual constructor` too

Reply:
I don't get it. It is an expect declaration. Every actual constructor is marked as actual of course

Reply:
sorry, nvm, mixed-up things - I thought that android's `actual` 😵‍💫